consensus/misc/flare_daemon.go

```diff
@@ -0,0 +1,24 @@
+// (c) 2019-2020, Flare Networks Limited.
+
+package misc
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/flare-foundation/coreth/core/state"
+	"github.com/flare-foundation/coreth/params"
+)
+
+var flareDaemonUpgrade = CoreContractUpdate{
```

Comment:
Regarding nomenclature, why one has `Upgrade` and the other has `Update` ? Sorry if I missed any previous conversation regarding this if any.